fix(acc): update steps with new menu - MTA-5898 (#4848)
scaleway/docs-content: pages/account/how-to/change-password.mdx

- A Scaleway account logged into the [console](https://console.scaleway.com)

> 1. Click your name on the top-right navigation menu to go to the [User Account](https://console.scaleway.com/account/security) page.

Reviewer: They should click the profile icon/photo, no?

> 1. Click your name on the top-right navigation menu to go to the [User Account](https://console.scaleway.com/account/security) page.

Reviewer 2: On peut faire les 2 clics en une étape, non ?
```suggestion
1. Click your profile icon and name on the top-right navigation menu to go to the [User Account](https://console.scaleway.com/account/security) page.
```

2. Click the **My Security** tab.
3. Scroll down to the **Change password** section and click the **Change password** button. A pop-up appears.

> 4. Enter your current password in the first box, then a new password in the second field.

Reviewer 3: ```suggestion
4. Enter your current password and your new password in the corresponding boxes
```
